rosidl_generator_py/cmake/rosidl_generator_py_generate_interfaces.cmake

```diff
@@ -96,15 +120,27 @@ add_dependencies(
 
 if(NOT rosidl_generate_interfaces_SKIP_INSTALL)
   if(NOT "${_generated_msg_files} " STREQUAL " ")
+    _ament_cmake_python_get_python_install_dir()
```

Comment:
Have you figured out a way to do this inside `.ament_python_install_package`? I've been digging, but couldn't find a solution.

In the worst case, we can just add a TODO and try to resolve it later, given that this PR is blocking the Python work.